### ISSUE TITLE
Button colors are not consistent

### ISSUE THREAD

**Issue by arielj** [fastruby/points#99]
https://ombulabs.atlassian.net/browse/DT-262

**Branch/Commit:**

Main branch

**Expected behavior:**

The color of the buttons with similar actions should be consistent. Buttons deleting stuff should be red-ish, buttons adding stuff should be green, other buttons should be yellow (just an example, not sure if those are the right decisions)

**Actual behavior:**

The colors are not consistent:
- we have a red button to bulk delete, but a green button to delete a singles story
- we have a red button to add a story, but a green button do add an estimate
- we have a green button to edit a story or an estimate, but a red button to edit the project
- we have a green button to edit a story, but the submit button is yellow, and the back button is yellow too

**Screenshots and Videos**

![image](https://user-images.githubusercontent.com/188464/137357722-16117d00-b7dd-4701-8512-4846b3630b63.png)
![image](https://user-images.githubusercontent.com/188464/137357824-0dd7b8eb-6f52-401b-b24a-b65cb226dc5f.png)

## EDIT:

After checking with nathalie, she gave some suggestions:
- the row actions are changing to something like this for now
![image](https://user-images.githubusercontent.com/188464/144893939-78231db6-1b21-46d7-b546-7f4f421b4814.png)

but she proposed something like this for the future:
![image](https://user-images.githubusercontent.com/188464/144895943-5a584332-c5b8-423d-8f5a-6611bd22eb41.png)
![image](https://user-images.githubusercontent.com/188464/144895959-b2076006-a998-4b97-919e-19acae9d08c0.png)

- the `Bulk Delete` button:
- - would be better if it looks like a link instead of a button so it doesn't look that important, (or maybe it shouldn't be visible until stories are selected? added by me)
- - the `Bulk Delete` button can be renamed to `Delete`, the `x story/ies` makes it clear it can be a bulk action
- - (added by me), maybe move bulk actions into a `more` dropdown? we could have bulk move / delete / clone for example

- the `Return to Projects Page` button should be a button, and shouldn't be there with the actions, it's a navigation action so it should be in the header or the nav bar, and should look like a button but like a link since it's navigation

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

**Comment by arielj** [comment]
hey! sure, sorry I missed this

I'm not an expert on color palletes so take this more as an idea and not a final decision. But I'd say:

In the list of stories for a project:
- `Delete` a story should be magenta like Bulk Delete
- `Add a story`, `Clone Project` and `Add Sub-Project` could be green
- `Edit` a story, `Edit Estimate` and `Edit or Delete project` could be yellow

In the forms:
- Primary action: green (Save Changes for example)
- Back button: yellow

Basically:
- creating/cloning things: green
- destroying/archiving: magenta
- the rest (edit/navigation/ambiguous): yellow

I think it's going to be more clear if it's ok after seeing the change in action.

The main problem I think it's with the `Delete` for a story that should be magenta for sure and with the `Add a Story` that should either be yellow or green but not magenta, the rest are kinda optional changes I'd say.

**Comment by JuanVqz** [comment]
I didn't cover the EDIT section in the description, seems like it belongs to another issue, what do you think?
